Makefile

```diff
@@ -1,20 +1,31 @@
 PRE_COMMIT_VERSION ?= 4.3.0
 
-.PHONY: init reinit check docs launch clean config
+.PHONY: init playwright test reinit check backup-database backup-file-storage backup restore-database restore-file-storage restore docs launch clean config
 
 init:
 	@test -d .venv || python3 -m venv .venv
 	.venv/bin/python -m pip install "pre-commit==$(PRE_COMMIT_VERSION)"
 	.venv/bin/pre-commit install
 
+playwright:
+	@test -d .venv || python3 -m venv .venv
```

Comment:
Should we use a dedicated venv for the tests? 🤔

Reply:
Made `playwright` depend on `init` and `test` depend on `playwright`.